aries-site/src/pages/templates/forms.js

```diff
@@ -29,11 +31,118 @@ const Forms = () => {
+            until this field has been completed.
+          </SubsectionText>
+        </Subsection>
+        <Subsection name="Using help text" level={3}>
+          <SubsectionText>
+            In most cases, the FormField label should be clear enough to
+            instruct the user how to complete a field. However, if help text is
+            required, ensure that it is kept brief and precise.
+          </SubsectionText>
+          <SubsectionText>
+            Help text may be relevant for cases such as{' '}
+            <Anchor label="password requirements" href="#sign-up" /> or{' '}
+            <Anchor label="additional resources" href="#customize" /> on a
+            field's context.
+          </SubsectionText>
+        </Subsection>
+        <Subsection name="Button placement and labeling" level={3}>
+          <SubsectionText>
+            Submit, reset, or cancel buttons should always be placed at the
+            bottom of a Form.
+          </SubsectionText>
+          <SubsectionText>
+            Button labels should be specific to the action of the button. For
+            example, a <Anchor label="sign-up form" href="#sign-up" /> may have
+            a submit button with label "Sign Up". Alternately,{' '}
+            <Anchor label="password change form" href="#change-password" /> may
+            have a button with label "Update Password".
+          </SubsectionText>
+          <SubsectionText>
+            Specifity in button labels gives the user confidence in the action
+            they are performing.
+          </SubsectionText>
+        </Subsection>
+        <Subsection name="Form submission" level={3}>
+          <SubsectionText>
+            Visual feedback is important to instill confidence in a user after
+            form submission.
+          </SubsectionText>
+          <Text weight="bold">On submission, your Form should:</Text>
+          <BulletedList
+            items={[
+              `Display any relevant errors (if applicable) with text that 
+              explains the necessary criteria to fix errors`,
+              `Provide user with indication that form was submitted 
+              successfully (notification, success page, etc.)`,
+              `Allow user to continue navigating through the application 
+              immediately`,
+            ]}
+          />
+          <SubsectionText>
+            Never force the user to wait for post-submission processing to
+            complete before they can continue using an application.
+          </SubsectionText>
+        </Subsection>
+        <Subsection name="Accessibility" level={3} gap="small">
+          <SubsectionText>
+            All Form input elements need to be contained within a FormField.
+          </SubsectionText>
+          <Text weight="bold">FormFields need to have:</Text>
+          <BulletedList
+            items={[
+              'An htmlFor property that matches the id of its child input',
```

Comment:
`htmlFor` is Grommet specific, resolving to label's `for` attribute in the DOM. I'm not sure which direction we want to go... provide general guidance vs. this guidance assumes Grommet...

Reply:
I will modify the copy to elaborate a little here.